modules/apps/document-library/document-library-preview-css/package.json

```diff
@@ -0,0 +1,15 @@
+{
+	"dependencies": {
+		"@clayui/css": "3.29.0"
+	},
```

Comment:
How much this dependency on claycss would be spread in this scenario of multiple CSS modules? May it have any implication when updating clay or patching?

Reply:
Not sure what you are asking but since we moved liferay-portal to a yarn monorepo (in `modules` directory) all dependencies in portal are managed (and deduplicated) there so, even if we add dependencies to `package.json` files in projects for correctness, we always make sure we ship the portal with a unique version of each.

Reply:
Ok that helps for sure. Still, In case we forget to update the version numbers in the corresponding `package.json` files in some project, as it is asking for an exact version, we'd end up with many clayui/css versions right? For the moment we have this only in 4 places and version is the same. I just wanted to highlight that this may increase the chances. 
On the other side, maybe our clay release process is taking this into account and there's nothing to worry about